Lib/collections/__init__.py

```diff
@@ -332,7 +332,8 @@ def namedtuple(typename, field_names, *, rename=False, module=None):
     if isinstance(field_names, str):
         field_names = field_names.replace(',', ' ').split()
     field_names = list(map(str, field_names))
-    typename = str(typename)
+    typename = _sys.intern(str(typename))
```

Comment:
This is not needed.

If `typename` is not valid name, ValueError will be raised below. But it already have seat in the interned string repository.

Reply:
The *typename* does not become automatically interned with `type(typename), (tuple,), namespace)`:

    >>> import sys
    >>> tn = 'Po'
    >>> tn1 = tn + 'int'
    >>> tn2 = tn + 'i' + 'nt'
    >>> t1 = type(tn1, (), {})
    >>> t2 = type(tn2, (), {})
    >>> t1.__name__ is t2.__name__
    False

So, I do think it it wise to intern the *typename*.

Reply:
Agree.